scripts/test-docker-ce-installation.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+set -euo pipefail
```

Comment:
Sometimes I like adding the x here to. Makes the output verbose, but you are thankful for it when things go wrong